Dockerfile

```diff
@@ -17,4 +17,6 @@ COPY --from=build ${DEPENDENCY}/META-INF /app/META-INF
 COPY --from=build ${DEPENDENCY}/BOOT-INF/classes /app
 COPY src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/SensorTypes.json src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/SensorTypes.json
 COPY src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/EffectorTypes.json src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/EffectorTypes.json
+COPY src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/FishTypes.json src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/FishTypes.json
```

Comment:
I think it would be more professional and effective to import a folder instead of file by file 🏎️